docs/en/week10/10-3.md

```diff
@@ -0,0 +1,310 @@
+---
+lang-ref: ch.10-3
+lecturer: Alfredo Canziani
+title: Transformer  Encoder-predictor-decoder architecture
+authors: Rahul Ahuja, jingshuai jiang 
+date: 15th April 2021
+---
+
+
+## The Transformer
+
+Expanding on our knowledge of attention in particular, we now interpret the fundamental building blocks of the transformer. In particular, we will take a forward pass through a basic transformer, and see how attention is used in the standard encoder-predictor-decoder paradigm.
+
+
+### Encoder-predictor-decoder architecture
+
+
+We should be familiar with the terminology of these modules from the previous lectures. 
+In the EBM latent variable architecture, we have $\vect{x}$ the observed input value which goes into a predictor. We have $\vect{y}$ which is the target value. The decoder modules is fed with a latent variable and output of the predictorn which outputs  $\tilda{y}$. $E$ is the energy function which minimises the energy between $\tilda{y}$ and $y$. 
+
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/ebm.png" style="zoom: 100%; background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> EBM model.
+</center>
+
+
+
+In Autoencoder structue , we observed there is no conditional input but only a target variable. It is fed through an encoder module which transforms into a hidden representation space, forcing only the most important information through.
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/autoencoder.png" style="zoom: 100%; background-color:#DCDCDC;" /><br>
+<b>Figure 2:</b> Autoencoder Architecture.
+</center>
+
+
+
+We will now take a look at these modules in the context of transformers.
+
+
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/transformer.png" style="zoom: 100%; background-color:#DCDCDC;" /><br>
+<b>Figure 3:</b> Transformer Architecture.
+</center>
+
+
+In a transformer, $\vect{y}$(target sentence) is a discrete time signal, which is  fed into a unit delay module succeeded by an encoder. The unit delay here transforms $y[j] —> y[j-1]$.  
```

Comment:
Missing a space after `$\vect{y}$`.
`->` is **not** LaTeX. Please, use LaTeX symbols.